tests/behat/features/prof/03-report/05-deputy-costs.feature

```diff
@@ -159,6 +159,14 @@ Feature: PROF deputy costs
     # previous=yes
     And the step with the following values CAN be submitted:
       | yes_no_profDeputyCostsHasPrevious_1 | yes |
+    And the step with the following values CANNOT be submitted:
+      | deputy_costs_previous_startDate_day   | 1    |
+      | deputy_costs_previous_startDate_month | 1    |
+      | deputy_costs_previous_startDate_year  | 2016 |
+      | deputy_costs_previous_endDate_day     | 1    |
+      | deputy_costs_previous_endDate_month   | 1    |
+      | deputy_costs_previous_endDate_year    | 2015 |
+      | deputy_costs_previous_amount          | 100  |
```

Comment:
We should probably test the boundary here by putting the end date 1 day before the start date, and in the happy path in the lines below change them to being on the same day.